crates/iota-graphql-rpc/src/data/pg.rs

```diff
@@ -64,6 +64,7 @@ impl QueryExecutor for PgExecutor {
         let max_cost = self.limits.max_db_query_cost;
         let instant = Instant::now();
         let pool = self.inner.get_pool();
+        #[allow(unexpected_cfgs)]
```

Comment:
Should these be `expect`?

Reply:
No, because it's not a universal warning, only for crates that don't have this feature.